### PR TITLE
release notes and property tables

### PR DESCRIPTION
**Pull request by danabauer**
1. update release notes to include `2024-05-16-beta.0` release
2. fix formatting on archived release notes
3. updates to guides, including property tables (partially finished)

### CODE REVIEW

File: docs/guides/buildings.mdx [hunk 1]
```diff
@@ -1,15 +1,82 @@
...
+| min_floor        | integer | 
+| facade_color     | varchar | 
+| facade_material  | varchar | 
+| roof_material    | varchar | 
+| roof_shape       | varchar | 
+| roof_direction   | double  | 
+| roof_orientation | varchar | 
+| roof_color       | varchar |  
+| eave_height      | double  | 
+| filename         | varchar |
```
Comment (jenningsanderson) [inline docs/guides/buildings.mdx:44]: Delete?

File: docs/guides/buildings.mdx [hunk 2]
```diff
@@ -1,15 +1,82 @@
...
+| min_floor        | integer | 
+| facade_color     | varchar | 
+| facade_material  | varchar | 
+| roof_material    | varchar | 
+| roof_shape       | varchar | 
+| roof_direction   | double  | 
+| roof_orientation | varchar | 
+| roof_color       | varchar |  
+| building_id      | varchar | 
+| filename         | varchar |
```
Comment (jenningsanderson) [inline docs/guides/buildings.mdx:72]: Delete?

File: docs/guides/buildings.mdx [hunk 3]
```diff
@@ -1,15 +1,82 @@
...
 ## Overview
 
-The Overture Maps `buildings` theme describes human-made structures with roofs or interior spaces that are permanently or semi-permanently in one place (source: [OSM building definition](https://wiki.openstreetmap.org/wiki/Key:building)).
+The Overture Maps `buildings` theme describes human-made structures with roofs or interior spaces that are permanently or semi-permanently in one place (source: [OSM building definition](https://wiki.openstreetmap.org/wiki/Key:building)). The theme includes two feature types:
+- **`building`**: The most basic form of a building feature. The geometry is expected to be the most outer footprint -- or roofprint if traced from satellite/aerial imagery -- of a building.
+- **`building_part`**: A single part of a building. Building parts may have the same properties as buildings, including color, material, and shape. A building part is associated with a parent building via a `building_id`, if its geometry is contained within the building footprint.
+
+The datasets are available as GeoParquet files with the following attributes: 
+<Tabs>
+<TabItem value="building" label="building" default>
```
Comment (jenningsanderson) [inline docs/guides/buildings.mdx:17]: Let's add a description column if we're going to include the tables?

Comment (danabauer) [reply docs/guides/buildings.mdx:19]: done!

**Pull request merged**